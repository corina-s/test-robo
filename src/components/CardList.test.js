import React from 'react';
import {shallow} from 'enzyme';
import CardList from './CardList';

it('expect to render Snapshot of Card Component', () => {
    const mockRobots = [
        {
            id: 1,
            name: 'Jon Snow',
            username: 'Jon',
            email: "j@j.com"
        }
    ]
    expect(shallow(<CardList robots={mockRobots}/>)).toMatchSnapshot();
})