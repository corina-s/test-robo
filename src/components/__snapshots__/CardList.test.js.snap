// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Snapshot of Card Component 1`] = `
<div>
  <Card
    email="j@j.com"
    id={1}
    key="0"
    name="Jon Snow"
  />
</div>
`;
